paf-mvp-demo-express/src/paf-client-node.ts

```diff
@@ -28,24 +27,18 @@ Ts8lo0jba/6zuFHUeRvvUN7o63lngkuhntqPXFiEVxAmxiQWVfFwFZ9F
 
 export const pafCmpApp = express();
 
-// This PAF client node only allows calls from the corresponding publisher's website
-const allowedOrigins = [getHttpsOriginFromHostName(pafDemoPublisherConfig.host)];
-
 addClientNodeEndpoints(
   pafCmpApp,
+  {
+    name: pafCmpConfig.name,
+    currentPublicKey: pafCmpPrivateConfig.currentPublicKey,
+    dpoEmailAddress: pafCmpPrivateConfig.dpoEmailAddress,
+    privacyPolicyUrl: new URL(pafCmpPrivateConfig.privacyPolicyUrl),
+  },
+  {
+    hostName: pafCmpConfig.host,
+    privateKey: pafCmpPrivateConfig.privateKey,
+  },
```

Comment:
Same concern here. The types aren't only for the compiler but also for the readers. :-)

Reply:
Same comment, I don't understand how you want to add explicit typing to a value passed to a method 🤔